src/components/Header/components/Login.tsx

```diff
@@ -1,113 +1,92 @@
-import React, { useEffect, useRef } from "react";
-import { Box, chakra, VStack } from "@chakra-ui/react";
-import FocusLock from "@chakra-ui/focus-lock";
+import React from "react";
+import { Box, chakra, useMultiStyleConfig, VStack } from "@chakra-ui/react";
 
-import Button from "../../Button/Button";
 import Icon from "../../Icons/Icon";
 import Link from "../../Link/Link";
 import List from "../../List/List";
 import Text from "../../Text/Text";
 import { LoginProps } from "./UpperNav";
 
-const Login = chakra(({ loginOpen, patronName, setLoginOpen }: LoginProps) => {
-  const greeting = useRef(null);
-
-  useEffect(() => {
-    if (patronName && loginOpen) {
-      greeting.current.focus();
-    }
-  }, [loginOpen, patronName]);
-
-  return (
-    <FocusLock isDisabled={!loginOpen}>
-      <Button
-        buttonType="link"
-        className={`withIcon ${
-          loginOpen
-            ? "closeLoginButton"
-            : patronName
-            ? "accountButton"
-            : "loginButton"
-        }`}
-        id="loginButton"
-        onClick={() => setLoginOpen(!loginOpen)}
-      >
-        {loginOpen ? "Close" : patronName ? "My Account" : "Log In"}
-        <Icon align="right" name={loginOpen ? "close" : "arrow"} size="small" />
-      </Button>
-      {loginOpen && (
-        <VStack className="loginMenu">
-          {patronName && (
-            <Box
-              id="patronGreeting"
-              data-testid="patronGreeting"
-              ref={greeting}
-              tabIndex={0}
+const Login = chakra(
```

Comment:
I wonder if we can namespace the components under `Header/components`.  While they aren't going to be exported, `Login` and `Mobile`, in particular, can be pretty confusing, and I think `Header.Login` or `NYPLHeader.Login`, etc, would make it clear that this shouldn't be used in trying to make a create a generic login component.